hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AMiscOperations.java

```diff
@@ -256,6 +277,7 @@ public void testS3AToStringUnitialized() throws Throwable {
   }
 
   /**
+<<<<<<< ours
```

Comment:
Looks like this came during merging. Can be removed later.

Reply:
probably best to revert and reapply. so as to avoid more confusion on cherrypicks